post/2023/beyond-sf/index.qmd

```diff
@@ -0,0 +1,305 @@
+---
+title: "R-spatial beyond `{sf}`"
+author: 
+  - name: Josiah Parry
+    affiliation: Esri
+    orcid: 0000-0001-9910-865X
+categories: 
+  - posts
+  - rstats
+tags: [rspatial]
+date: "2023-09-06"
+slug: r-spatial-beyond-sf
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+**sf** has transformed the R ecosystem and helped pave the way for wider adoption of geospatial workflows by R users. The R-spatial ecosystem, however, is much bigger than just sf. There are lower level packages such as **geos**, **wk**, and **s2** that provide access to geometries and algorithms outside of the context of sf. These packages are often more performant than sf but require a different frame of mind when using them.
+
+The goal of this post is to demystify the way in which sf represents geometries and explain how other geometry libraries can be used along side their typical sf workflows.
+
+## Understanding `sf`
+
+Most R users that do geospatial analysis are familiar with sf and understand spatial workflows in that context only. Many users of `sf` view it as this magical data frame that lets you do spatial analysis—and that is exactly how it feels! This means that sf has done a great job in making spatial analysis feel a lot easier for the vast majority of R users. 
```

Comment:
```suggestion
Most R users that do geospatial analysis are familiar with **sf** and understand spatial workflows in that context only. Many users of **sf** view it as this magical data frame that lets you do spatial analysis -- and that is exactly how it feels! This means that **sf** has done a great job in making spatial analysis feel a lot easier for the vast majority of R users. 
```

Reply:
Is there a reason to get rid of the em-dash in favor of two hyphens? Typically an em-dash does not have a space before or after it.

Reply:
Addressed bolding. Please unresolve if current em-dashes are unwanted.

Reply:
> Is there a reason to get rid of the em-dash in favor of two hyphens? Typically an em-dash does not have a space before or after it.

Not really. I do not know how the em-dash will render so I replaced that with the style I know. You can keep em-dash if you think that's better.